app/src/main/java/ch/epfl/skysync/screens/Calendar.kt

```diff
@@ -69,7 +69,7 @@ fun CalendarScreen(
                 flightCalendar.getFirstFlightByDate(date, time)
               },
               onFlightClick = { selectedFlight ->
-                navController.navigate(Route.FLIGHT_DETAILS + "/${selectedFlight}")
+                // navController.navigate(Route.FLIGHT_DETAILS + "/${selectedFlight}")

```

Comment:
Why ?

Reply:
we don't have a crew specific screen for the flight details yet. If we leave activated a crew will be able to modify and delete a flight.